lumibot/brokers/broker.py

```
@@ -2635,15 +2635,117 @@ def modify_order(self, order, stop_price: Union[float, None] = None, limit_price
 
     def submit_order(self, order) -> Order:
         """Conform an order for an asset to broker constraints and submit it."""
+        self.resolve_option_order_intent(order)
         self._conform_order(order)
         return self._submit_order(order)
 
+    def resolve_option_order_intent(
```

Pylint (4.0.7)

[refactor] Too many local variables (18/15)

(R0914)